spec/features/landing_phone_or_online_filter_for_other_advice_types_spec.rb

```diff
@@ -22,21 +22,15 @@ def given_firms_with_advisers_were_previously_indexed
       create(:other_advice_method, name: 'Advice online (e.g. by video call / conference / email)', order: 2)
     ]
 
-    in_person_advice_methods = create_list(:in_person_advice_method, 3)
-
     with_fresh_index! do
-      @first = create(:firm_with_no_business_split, retirement_income_products_flag: true, wills_and_probate_flag: true, other_flag: true, in_person_advice_methods: [], other_advice_methods: other_advice_methods)
+      @first = create(:firm_with_no_business_split, retirement_income_products_flag: true, wills_and_probate_flag: true, in_person_advice_methods: [], other_advice_methods: other_advice_methods)
       @glasgow = create(:adviser, firm: @first, latitude: 55.856191, longitude: -4.247082)
 
-      @second = create(:firm_with_no_business_split, retirement_income_products_flag: true, wills_and_probate_flag: true, other_flag: true, in_person_advice_methods: [], other_advice_methods: other_advice_methods)
+      @second = create(:firm_with_no_business_split, retirement_income_products_flag: true, wills_and_probate_flag: true, in_person_advice_methods: [], other_advice_methods: other_advice_methods)
       @leicester = create(:adviser, firm: @second, latitude: 52.633013, longitude: -1.131257)
 
-      @excluded = create(:firm_with_no_business_split, retirement_income_products_flag: true, other_flag: true, in_person_advice_methods: [], other_advice_methods: other_advice_methods)
+      @excluded = create(:firm_with_no_business_split, retirement_income_products_flag: true, in_person_advice_methods: [], other_advice_methods: other_advice_methods)
       create(:adviser, firm: @excluded, latitude: 51.428473, longitude: -0.943616)
-
-      create(:firm_with_no_business_split, other_flag: true, in_person_advice_methods: in_person_advice_methods, other_advice_methods: other_advice_methods) do |f|
```

Comment:
I think it's affect was something to do with the number of result returned. If something was wrong an extra result (this one) would appear and so it would break a results count test somewhere. What trying to say is, it might be here to ensure it never gets seen by accident.